docs/DeveloperGuide.md

```diff
@@ -114,7 +114,8 @@ Here are the other classes in `Logic` (omitted from the class diagram above) tha
 
 How the parsing works:
 
-- When called upon to parse a user command, the `AddressBookParser` class creates an `XYZCommandParser` (`XYZ` is a placeholder for the specific command name e.g., `AddCommandParser`) which uses the other classes shown above to parse the user command and create a `XYZCommand` object (e.g., `AddCommand`) which the `AddressBookParser` returns back as a `Command` object.
+- When called upon to parse a user command, the `ModBookParser` class creates an `XYZCommandParser` (`XYZ` is a placeholder for the specific command name e.g., `AddCommandParser`) which uses the other classes shown above to parse the user command and create a `XYZCommand` object (e.g., `DetailCommand`) which the `ModBookParser` returns back as a `Command` object.
+- The name of the `Command` Object may also include the type of object the command is acting on depending on the command. For example, the add command will return either an `addModCommand`, `addLessonCommand` or `addExamCommand` object.
```

Comment:
perhaps can rephrase to "The name of the `Command` Object may also include the type of object the command is acting on. For example, `add mod` command returns a ..., `add lesson` command returns a...."